tutorial/consuming_packages/the_flexibility_of_conanfile_py.rst

```diff
@@ -202,6 +202,7 @@ platform without adding more changes.
                 self.folders.generators = os.path.join("build", "generators")
```

Comment:
Check this comment: https://github.com/conan-io/examples2/pull/119/files#r1374419599